code/game/objects/items.dm

```diff
@@ -1017,6 +1017,11 @@ var/global/list/image/blood_overlays = list()
 	if(flags & INVULNERABLE)
 		return
 	spawn(0) //this is needed or multiple items will be thrown sequentially and not simultaneously
+		if(anchored)
```

Comment:
Yes but only singularitys at stage 5 de-anchor objects, currently any size singularity will move anchored items, but not de-anchor them